code/scripting/lua/LuaUtil.h

```diff
@@ -91,6 +91,14 @@ void tableToList(LuaTable& table, Container& list) {
 }
 
 const char* getValueName(ValueType type);
+
+/**
+ * Processes a LuaValue and returns a vec3d.  The LuaValue could be a Vector, an array with three elements, or a table with x/X, y/Y, and z/Z entries.
+ * 
+ * Throws a LuaException if the conversion was not successful.
+ */
```

Comment:
It's a bit verbose to put in the script docs, which I might otherwise prefer unless you disagree, so it might be worth putting here at least excerpts of the exact expected syntax, e.g.
`ba.createVector(0.0)`
`{0.0, 0.0, 0.0}`
`{'x': 0.0, 'y': 0.0, 'z': 0.0}`